lib/krikri/parsers/json_parser.rb

```diff
@@ -44,7 +44,17 @@ def values?
 
       private
 
+      ##
+      # @see Krikri::Parser#get_child_nodes
+      #
+      # @param name_exp [String]  Object property name
+      # @return [Krikri::Parser::ValueArray]
       def get_child_nodes(name)
+        return child_node(name) if @node.include?(name)
+        Krikri::Parser::ValueArray.new([])  # nothing found
```

Comment:
Can you say something quickly about why this change is necessary? Does `Krikri::Parser::ValueArray.new(vals)` not come up with `Krikri::Parser::ValueArray.new([])` when nothing is found, below?